viewing_party/party.py

```diff
@@ -1,23 +1,220 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movies_dict = {}
+    if not title or not genre or not rating:
+        return None
+    else:
+        movies_dict["title"] = title
+        movies_dict["genre"] = genre
+        movies_dict["rating"] = rating 
+    return movies_dict
+
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    is_valid = True
+
+    ### checking that watched_list is not empty
+    if watched_list:
+    ### checking that title is not already in watched_list keys
+        for movie_dict in watched_list:
+            if movie_dict["title"] == movie["title"]:
+                is_valid = False

```

Comment:
You could use the `break` keyword here in order to stop the for loop, that way loop doesn't continue to run after you have confirmed that the movie is already inside of their `watched_list`.